CONTRIBUTING.md

```diff
@@ -12,21 +12,21 @@ If you've found a bug that hasn't been reported yet you can help us by [creating
 
 ### Providing translations
 
-Skosmos core languages (English, Finnish and Swedish) are maintained by us. In addition to these three languages volunteers have provided translations for other languages on the transifex platform. We are happy to receive both translations for new languages as well as **updates to the existing translations**. Translations included in the 1.7 release include English, Chinese, Spanish, French, German, Italian, Finnish, Swedish and Norwegian (both Bokmål & Nynorsk).
+Skosmos core languages (English, Finnish and Swedish) are maintained by us. In addition to these three languages volunteers have provided translations for other languages on the transifex platform. We are happy to receive both translations for new languages as well as **updates to the existing translations**. Skosmos has been translated to more than 10 languages by members of the community.
```

Comment:
>Skosmos has been translated to more than 10 languages by members of the community.

:tada: 

>are maintained by us.

Maybe clarify who "us" is, by placing it in the third person, like "are maintained by the Skosmos team", or Skosmos core team, or NatlibFi, etc.